aosp_diff/caas_cfc/frameworks/base/0004-WA-for-rotation-framework.patch

```diff
@@ -0,0 +1,114 @@
+From 2713d08ab434c8e16f6d7d1f15fc91efd7e8b119 Mon Sep 17 00:00:00 2001
+From: gkdeepa <g.k.deepa@intel.com>
+Date: Thu, 28 Oct 2021 13:41:35 +0530
+Subject: [PATCH] framework logs and WA
+
+Tracked-On:
+---
+ core/jni/android_hardware_Camera.cpp | 33 +++++++++++++++++++++++-----
+ 1 file changed, 28 insertions(+), 5 deletions(-)
+
+diff --git a/core/jni/android_hardware_Camera.cpp b/core/jni/android_hardware_Camera.cpp
+index bc69735d7453..d46c77396960 100644
+--- a/core/jni/android_hardware_Camera.cpp
++++ b/core/jni/android_hardware_Camera.cpp
+@@ -40,7 +40,7 @@ enum {
+     // Keep up to date with Camera.java
+     CAMERA_HAL_API_VERSION_NORMAL_CONNECT = -2,
+ };
+-
++int checkflag=0;
+ struct fields_t {
+     jfieldID    context;
+     jfieldID    facing;
+@@ -544,9 +544,14 @@ static void android_hardware_Camera_getCameraInfo(JNIEnv *env, jobject thiz,
+         jniThrowRuntimeException(env, "Fail to get camera info");
+         return;
+     }
++    if(checkflag == 1){
++       if(cameraInfo.facing == 1) 
++        env->SetIntField(info_obj, fields.orientation, 270);
```

Comment:
Take Classin as one example, when the front camera is called, then we need "SetIntFied", but when it is back, we don't need call "SetIntFied"?